jwplayer-appletv-web-app/resources/configs/VCyJXbpY/config.json

```diff
@@ -1,11 +1,25 @@
 {
   "playlists": [
-    "K6Sl8yPJ",
-    "BXatQw7p",
-	  "E12RS6r7"
+    {
+      "playlistId": "K6Sl8yPJ",
+      "autoAdvance": true,
```

Comment:
@RikHeijdens I think it's more appropriate for `autoAdvance` and `autoAdvanceWarningOffset` to be global settings for the application, rather than a per-playlist setting.